src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm.hpp

```diff
@@ -120,11 +120,11 @@ struct brgemm_ref_kernel : public dnnl::impl::cpu::x64::brgemm_kernel_t {
         return nullptr;
     }
     [[nodiscard]] const dnnl::impl::cpu::x64::brgemm_desc_t& get_brg() const override {
+        static const dnnl::impl::cpu::x64::brgemm_desc_t brg;
```

Comment:
If it's used nowhere, can we replace this line with `OV_CPU_JIT_EMITTER_THROW("get_brg should not be called for reference kernel");` as it's done for the `get_jit_generator()` above? This is just simple debug kernel :)

Reply:
This is a good idea, but still formally we have to return a correct reference.